_component_design/radio-buttons.md

```diff
@@ -7,157 +7,140 @@ usage: >
+- Keep the number of options to between 2 and 5 choices.
+- There is always a default option selected (preferably at the top or the list).
+- If the user clicks on another option, the first is unselected.
+- Radio buttons should only be used when the list can be vertical. If space is not available, consider a dropdown menu.
+- List options in an order that makes logical sense. Depending on the context, the order could be alphabetical or most common to least common. For example, if listing accounts by name, list them alphabetically. In billing, paying the amount due should be listed above letting the user enter a custom payment amount.
+- Use checkboxes and radio buttons only to change settings, not as action buttons that make something happen.
+
 {% endcolumn %}
-{% column right:"hxCol" %}
-
-## Error
-
-<div class="ui form error">
-  <div class="grouped fields required">
-    <label>Select an option</label>
-    <div class="field error checked">
-      <div class="ui radio checkbox">
-        <input type="radio" name="radio3" check="checked">
-        <label>Option 1</label>
-      </div>
-    </div>
-    <div class="field error">
-      <div class="ui radio checkbox">
-        <input type="radio" name="radio3">
-        <label>Option 2</label>
-      </div>
-    </div>
-    <div class="field error">
-      <div class="ui radio checkbox">
-        <input type="radio" name="radio3">
-        <label>Option 3</label>
-      </div>
-    </div>
-  </div>
-  <div class="ui error message">
-    There is an error with the response to this question.
-  </div>
+
 </div>
+
+## Specifications
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Composition
+
+A radio button group consists of a Section Label, followed by pairs of radio buttons and radio button labels.
+
+#### Labeling:
+
+- Labels should start with a capital letter.
+- Labels should not have punctuation at the end, such as a period, unless more than one sentence is used. 
+
 {% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Radio button group composition"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/radio-buttons/radio-composition.svg)
+{% endfigure %}
+{% endcolumn %}
+
 </div>
 
-# Radio Button Group
+### Spacing
 
-Use when selecting a **single item** from a set if it's important for the user
-to see all options at once. This alternate can be used instead of regular radio
-buttons if the options are **simple**, limited in number and individual length.
+<div class="hxRow">
 
-## Enabled
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+A title should always be located above the radio buttons. More context can be provided under the title (subtitle) but it is optional (shown here as “Select an option” on left). If subtitle is not needed, use specs in the design on the right.
```

Comment:
I don't understand "use specs in the design on the right"

Reply:
removed.